bioimageio/spec/__init__.py

```diff
@@ -1,4 +1,23 @@
 from . import v0_1, v0_3
-from .latest import *
+from .build_spec import build_spec
 
-__version__ = v0_3.schema.get_args(FormatVersion)[-1]
+# autogen: start
+from bioimageio.spec.shared import fields
+from . import nodes, raw_nodes, schema, utils
+from .converters import maybe_convert_manifest, maybe_convert_model
+from .raw_nodes import FormatVersion
+
+fields = fields
+
+
+get_nn_instance = utils.get_nn_instance
+download_uri_to_local_path = utils.download_uri_to_local_path
+load_raw_model = utils.load_raw_model
+load_model = utils.load_model
+
+# autogen: stop
+
+# assuming schema will always be part of spec
+from . import schema
+from .raw_nodes import FormatVersion
+__version__ = schema.get_args(FormatVersion)[-1]
```

Comment:
here you are actually assuming that schema always imports `typing.get_args`

Did you avoid 
```python
try:
    from typing import get_args
except ImportError:
    from typing_extensions import get_args

```
her ein __init__.py on purpose?

Reply:
good catch. this doesn't make any sense - I'm going to move that import to `spec.shared.common`, and change all imports accordingly.